weave/trace/weave_client.py

```diff
@@ -722,8 +723,7 @@ def finish_call(
             postprocessed_output = original_output
         self._save_nested_objects(postprocessed_output)
 
-        output = map_to_refs(postprocessed_output)
-        call.output = output
+        call.output = map_to_refs(postprocessed_output)
```

Comment:
I found it a little confusing to be redefining the `output` parameter to `finish_call`. Most code uses the `original_output` alias, small update for this change below.

Reply:
makes sense